condax/__init__.py

```diff
@@ -1 +1,8 @@
-__version__ = "0.0.5"
+import sys
+
+if sys.version_info < (3, 8):
+    from importlib_metadata import version
+else:
+    from importlib.metadata import version
+
+__version__ = version(__package__)
```

Comment:
Doing this at the root imposes a startup time cost for CLIs. I'd recommend https://github.com/ofek/hatch-vcs#build-hook to save to a file which you would then import here

Reply:
Thanks for the suggestion! Let's add this as a future todo for both condax  and conda-lock.

I honestly missed this in the `hatch-vcs` README because there's no complete example for it (just for the table header), making it blend in with "advanced options which I probably don't need".